### DISCUSSION TITLE
Noob questions

### DISCUSSION THREAD

**Discussion by m0rg5** [vpnhood/VpnHood#30] [Q&A] [answered]
I'm reading through, am not a dev, and what VPNhood is, is still unclear to me.

In simple terms: you're obscuring my traffic by redirecting traffic over a secure connection (default IP 51.81.81*)?
Where is this IP come from? Who sees the data? How is it encrypted?

How do I add my own IP, or existing OpenVPN config? (what I searched for that led me here)

**Comment by trudyhood** [comment, marked as answer]
_Where is this IP come from?_
The IP is one of our public servers for test purposes to see how it works without setting up your server.

_Who sees the data?_
Anyone who runs a VPN server can see your destination IP, destination domains, and data that is not encrypted. Nowadays, all data have been already encrypted, but the VPN server runner can still see the destination IP and domain if they want to. For that, you can only trust the VPN server or run your server with an open-source server you trust.
We don't monitor and record the destination IP and domain, but who knows about the hosting provider! They claim not, our public servers hosted by OVH.
VpnHood server doesn't try to extract destination domain too, but theoretically it is possible.

_How is it encrypted?_
We are using SSL. Many VPNs redirect and encrypt the traffics to the server. VpnHood does the same but also tries to make the encrypted channel look precisely the same as an ordinary website, so firewalls read the data and can't find it is a VPN. They see it as regular websites.

_How do I add my own IP, or existing OpenVPN config?_
VpnHood has its protocol; you need to run your VpnHood server and VpnHood client. Honestly, if OpenVPN works in your situation, you may don't need VpnHood :)
See our Wiki page if you want to set up your own server and contact us if you have any problem
https://github.com/vpnhood/VpnHood/wiki
